test/functional/inspec_schema_test.rb

```diff
@@ -0,0 +1,21 @@
+# encoding: utf-8
+
+require 'functional/helper'
+
+describe 'inspec schema' do
+  include FunctionalHelper
+
+  describe 'extracting the platforms schema export' do
+    it 'can execute a platforms schema command' do
+      out = inspec('schema platforms')
+      out.stderr.must_equal ''
+      out.exit_status.must_equal 0
+      data = JSON.parse(out.stdout)
+      data.class.must_equal Array
+      data.size.must_be :>, 1
```

Comment:
Not sure how we officially feel about multiple arguments not being in `()`...but I prefer it.